### PR TITLE
add food security discovery

### PR DESCRIPTION
**Pull request by aboydnw**
for the agriculture thematic area

### CODE REVIEW

File: discoveries/agriculture-food-security-east-africa.discoveries.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+   *The Famine Early Warning Systems Network (FEWS NET) was created in 1985 by the [U.S. Agency for International Development](https://www.usaid.gov/what-we-do/agriculture-and-food-security) (USAID). FEWS NET assembles global and regional analyses of food security (particularly conditions for farming and livestock husbandry) to help governments and relief agencies plan for and respond to humanitarian crises. Several U.S. agencies support FEWS NET; NASA provides satellite imagery and climate, weather, and hydrologic data.
+   
+   NASA Earth Observatory maps by Lauren Dauphin, using Acute Food Insecurity Phase data from the [Famine Early Warning Systems Network](https://fews.net/horn-africa) and rainfall accumulation data from [Climate Hazards Group InfraRed Precipitation with Stations](https://earlywarning.usgs.gov/fews/product/598) (CHIRPS). NASA animations by Trent Schindler, NASA [Scientific Visualization Studio](https://svs.gsfc.nasa.gov/), using data from the [FEWS NET Land Data Assimilation System](https://ldas.gsfc.nasa.gov/fldas/models/forecast) (FLDAS). 
+   
+   Story by [Michael Carlowicz](https://earthobservatory.nasa.gov/about/michael-carlowicz).*
```
Comment (danielfdsilva) [inline discoveries/agriculture-food-security-east-africa.discoveries.mdx:78]: What is the purpose with the `*`?

**Review by danielfdsilva — CHANGES_REQUESTED** [review]
Nice to see another story!

Couple of things:
- The cover image is very heavy. Try to run it through http://compressor.io/ to see if it helps. You can also probably reduce the size, but never below 1920.
- The gif is also pretty big. Compressor may help, but with the animation there may not be a lot to do.
- If it makes sense, consider using the image blocks with images on the sides, as it gives a more interesting flow to the story.

Let me know if you need help

**Comment by aboydnw** [comment]
Alright I think I am lost haha. I pushed a couple changes based on your comments @danielfdsilva but I don't see them reflected here. When I view the git log this is what I see:
image
I'm not sure if that's correct, but the origin is a few behind my recent changes. Any ideas what I am doing wrong?

**Comment by aboydnw** [comment]
okay @danielfdsilva now it is up to date! :) ready for second review

**Review by danielfdsilva — APPROVED** [review]
@aboydnw Decreased the cover image to 2500px wide and added a content fix. Have a look at the last commit.

Otherwise all good.

**Pull request merged**